cli/provider/service.go

```diff
@@ -83,7 +85,21 @@ func (n *ServiceProvider) GetService(ctx context.Context, cmd string) (interface
 	case "config", "update":
 		return tools.NewTools(n.logger, tel), nil
 	// TODO: add case for mock
-	case "record", "test", "mock", "normalize":
+	case "record", "test", "mock":
+		// Check if the config file exists on the path or not and if it does not, we create it.
+		if !utils.CheckFileExists("keploy.yml") {
+			toolsService := tools.NewTools(n.logger, tel)
+			n.cfg.Path = strings.TrimSuffix(n.cfg.Path, "/keploy")
+			yamlData, err := yaml.Marshal(n.cfg)
+			n.cfg.Path = n.cfg.Path + "/keploy"
+			if err != nil {
+				n.logger.Debug("failed to marshal the config")
+			}
+			err = toolsService.CreateConfig(ctx, "keploy.yml", string(yamlData))
```

Comment:
Can you provide a screenshot by using 3 flags like selected test-sets etc and generate the config file..?